tutorials/Additive_and_multiplicative_bias.ipynb

```diff
@@ -964,16 +968,16 @@
     "import matplotlib.pyplot as plt\n",
     "fcst_flow = fcst.temp_scrn\n",
     "obs_flow = obs.temp_scrn\n",
-    "# Calculate additive bias precentage and preserve the \"lat\" dimension\n",
+    "# Calculate percent bias and preserve the \"lat\" dimension\n",
```

Comment:
@durgals - I changed "additive bias percentage" to "percent bias". Was it correct to make this change?

Reply:
@Steph-Chong . Yes.